class/prometheus-pushgateway.yml

```diff
@@ -9,4 +15,14 @@ parameters:
           - prometheus-pushgateway/component/main.jsonnet
         input_type: jsonnet
         output_path: prometheus-pushgateway/
-
+        output_type: yaml
+      - input_paths:
+          - prometheus-pushgateway/helmcharts/prometheus-pushgateway
+        input_type: helm
+        output_path: prometheus-pushgateway/01_helmchart
+        helm_params:
+          release_name: platform
+          namespace: '${prometheus_pushgateway:namespace}'
+        helm_values:
```

Comment:
It might make sense to expose the full `helm_values` in the `defaults.config` to be able to configure it further via the hierarchy?